api/README.md

```diff
@@ -234,17 +234,17 @@ How to Run the Full System (Two Services)
 This powers the actual meta-agent chat orchestration (TaskInterpreter, AgentYAMLBuilder, WorkflowYAMLBuilder).
 
 ```
-maestro serve ./demos/workflows/meta-agents-v2/SELECTED_META_FILE/agents.yaml ./demos/workflows/meta-agents-v2/SELECTED_META_FILE/workflow.yaml
+maestro serve ./workflows/meta-agents-v2/SELECTED_META_FILE/agents.yaml ./workflows/meta-agents-v2/SELECTED_META_FILE/workflow.yaml
 ```
 
 So to create the agents.yaml: 
 ```
-maestro serve ./demos/workflows/meta-agents-v2/agents_file_generation/agents.yaml ./demos/workflows/meta-agents-v2/agents_file_generation/workflow.yaml
+maestro serve ./workflows/meta-agents-v2/agents_file_generation/agents.yaml ./workflows/meta-agents-v2/agents_file_generation/workflow.yaml
```

Comment:
Prefix of `./../maestro-demos` may be useful here as well.

Reply:
see previous